src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/integration/MovePriceExceptionTest.kt

```diff
@@ -64,7 +67,7 @@ internal class MovePriceExceptionTest : IntegrationTest() {
 
     isAtPage(UpdatePrice)
       .assertTextIsPresent("Existing price exceptions")
-      .isRowPresent<UpdatePricePage>("August", year.value, "£3000.00")
+      .isRowPresent<UpdatePricePage>(month.name.titleCased(), year.value, "£3000.00")
```

Comment:
Resolving a bug I found as a result of real calendar month change (integration tests started failing).  This should not have been hard coded.